alpha/template/composite/builder.go

```diff
@@ -268,13 +261,12 @@ func (cb *CustomBuilder) Build(ctx context.Context, reg image.Registry, dir stri
 	}
 	// build the command to execute
 	cmd := exec.Command(customConfig.Command, customConfig.Args...)
-	cmd.Dir = cb.builderCfg.ContainerCfg.WorkingDir
 
 	// custom template should output a valid FBC to STDOUT so we can
 	// build the FBC just like all the other templates.
-	v, err := cmd.Output()
+	v, err := cmd.CombinedOutput()
```

Comment:
Fair.  The origin of this change was when this version of opm would shell out to a specified containerized opm version in a manner that caused that binary to generate a usage statement and quit with a zero exit status.  That meant that we discarded all remediation data by the time we tried to parse the data (and failed), so capturing the STDERR was the only way to be sure. 

I think it's likely more beneficial to service the more likely use-case of testing for error status rather than fix a (now obsolete) pretty obscure edge case.